OEM/Z370 AORUS Gaming 7/config.plist

```diff
@@ -105,7 +105,7 @@
   <key>Boot</key>
   <dict>
     <key>Arguments</key>
-    <string>ngfxcompat=1</string>
+    <string>ngfxcompat=1 -v</string>
```

Comment:
The `verbose` flag I'm using for testing purposes only. If you need to verbose output you can choose some flags in Clover. When you select boot entry and press `space-bar` you can edit boot args.